subprojects/shipkit/src/main/groovy/org/shipkit/internal/gradle/java/tasks/DownloadPreviousPublications.java

```diff
@@ -25,7 +25,9 @@ private void downloadRemoteFile(String remoteUrl, File localFile) {
             IOUtil.downloadToFile(remoteUrl, localFile);
         } catch (Exception e) {
             LOG.lifecycle("  Unable to download, ignoring. Run with '-d' for stack trace. Url: {}", remoteUrl);
-            LOG.debug("Unable to download, ignoring", e);
+            LOG.debug("Unable to download, ignoring." +
```

Comment:
This should probably be a "lifecycle" message, right?

Suggestion: 
 - lifecycle message with general info + a hint how to resolve the issue
 - debug message with stack trace